sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -1794,7 +1794,7 @@ BRACKETED_EMPTY_COMMENT
     ;
 
 BRACKETED_COMMENT
-    : '/*' ~[+] .*? '*/' -> channel(HIDDEN)
+    : '/*' ~[+] ( ~'/' | ~'*' '/' ~'*' )*? BRACKETED_COMMENT? ( ~'/' | ~'*' '/' ~'*' )*? '*/' -> channel(HIDDEN)
```

Comment:
`    : '/*' (BRACKETED_COMMENT|.)*? '*/' -> channel(HIDDEN)`?

Reply:
This not works well.

Reply:
how about 
```
'/*' ~[+] ~[/]*? BRACKETED_COMMENT? ~[/]*? '*/' -> channel(HIDDEN)
```